src/Session.cpp

```diff
@@ -369,7 +373,9 @@ bool CSession::InitializeDRM(bool addDefaultKID /* = false */)
 
     LOG::Log(LOGDEBUG, "Entering encryption section");
 
-    if (licenseKey.empty())
+    std::string_view licenseType = CSrvBroker::GetKodiProps().GetLicenseType();
+
+    if (licenseKey.empty() && licenseType != "org.w3.clearkey")
```

Comment:
instead to add a specific check for clearkey here,
i think its more appropriate that each DRM that require licenseKey
check if it is empty and print an error (more likely on OpenDRM method), otherwise it is simply ignored

Reply:
I've changed this to your suggestion, I think it's the right way to go.